app/views/jst/courses/roster/rosterUser.handlebars

```diff
@@ -2,7 +2,11 @@
   {{>avatar}}
 </td>
 <td>
-  <a href="{{html_url}}" class="roster_user_name">{{name}}</a>
+  {{#if canViewSisId}}
+  <a href="{{html_url}}" class="roster_user_name">{{name}}</a> ({{short_name}})
```

Comment:
What actual needs to happen here is that if name and short_name are the same then the (shortname) should not be shown. There is no equality comparison in handbars so I am not sure how to catch that with out changing what was is sent in the api.